lib/cocina/models/descriptive_basic_value.rb

```diff
@@ -13,9 +13,14 @@ class DescriptiveBasicValue < Struct
       attribute :code, Types::Strict::String.meta(omittable: true)
       # URI value of the descriptive element.
       attribute :uri, Types::Strict::String.meta(omittable: true)
-      attribute :standard, Types::Strict::Array.of(Types::Strict::String).meta(omittable: true)
-      attribute :encoding, Types::Strict::Array.of(Types::Strict::String).meta(omittable: true)
+      attribute :standard, Standard.optional.meta(omittable: true)
+      attribute :encoding, Standard.optional.meta(omittable: true)
```

Comment:
not backwards compatible - be prepared for problematic data?

Reply:
we aren't using this yet, NBD.